lib/sig.js

```diff
@@ -113,7 +113,7 @@ var actionsObjRegexp = new RegExp(
     jsKeyStr + sliceStr   + '|' +
     jsKeyStr + spliceStr  + '|' +
     jsKeyStr + swapStr +
-  '),?\\n?)+)\\};'
+  '),?(\\r\\n|\\r|\\n)?)+)\\};'
```

Comment:
I ran the tests on Windows 10 and failed. Then when I debug the code I found the 'body' in function extractActions in sig.js has all newline as \\r\\n. So I think this is caused by Windows.